documentation/development/steps_list.md

```diff
@@ -2,11 +2,24 @@
 
 _The following refers to all commands except `git-new-pull-request`, `git-repo` and `git-town`._
 
+The individual steps that each Git Town command executes are represested via the
+[command pattern](https://en.wikipedia.org/wiki/Command_pattern).
+This allows allow for robust and fully automated `--continue`, `--abort`,
+and `--undo` functionality.
+To distinguish the command-pattern commands from the Git Town commands in [src/cmd](../../src/cmd),
+we'll call the former `steps` from now on.
+
 Each Git Town command begins by inspecting the current state of the Git repository
 (which branch you are on, whether you have open changes).
 If there are no errors, it generates a list of steps to run.
-Each step is a go `struct`.
-
+Steps are in [src/steps](../../src/steps)
+and implement the individual steps that each Git Town command performs,
+like for example [changing to a different Git branch](../../src/steps/checkout_branch_step.go)
+or [pulling down updates for the current branch](../../src/steps/pull_branch_step.go).
+
+Steps are Go structs that have a `Run` method to execute the step
```

Comment:
`Steps are Go structs that have a `Run` method to execute the step` belongs in the previous paragraph

Reply:
Good catch, done